js/src/elements/form-behavior.ts

```diff
@@ -384,8 +384,15 @@ export class FormBehaviorElement extends HTMLElement {
 				continue;
 			}
 
-			// Don't check markdown editor attachment input
-			if (input.id === 'attachment-count') {
+			if (input.hasAttribute('data-skip-validation')) {
```

Comment:
Still saw inline error after adding skip attribute, expected? 

<img width="495" alt="image" src="https://user-images.githubusercontent.com/30843002/186240994-fb6ff344-5ef2-4a5d-983e-4d3d0951a423.png">

Reply:
Added a line to clear the error message. Please try it again.